src/Features/Core/Portable/CodeFixes/Configuration/ConfigureSeverity/ConfigureSeverityLevelCodeFixProvider.cs

```diff
@@ -24,11 +24,11 @@ internal sealed partial class ConfigureSeverityLevelCodeFixProvider : IConfigura
     {
         private static readonly ImmutableArray<(string name, string value)> s_editorConfigSeverityStrings =
```

Comment:
The tuple elements "name" and "value" here seems quite confusing -- if anything I would have expected them to be flipped around where the value is the one being put into the file. Maybe call one "display string" or something so it's very clear it's for display only?